core/src/impl/Kokkos_Core.cpp

```diff
@@ -690,6 +690,8 @@ void pre_initialize_internal(const Kokkos::InitializationSettings& settings) {
 #elif defined(KOKKOS_ARCH_AMD_ZEN3)
   declare_configuration_metadata("architecture", "CPU architecture",
                                  "AMD_ZEN3");
+#elif defined(KOKKOS_ARCH_RISCV_MILKV)
+  declare_configuration_metadata("architecture", "CPU architecture", "RISCV")
```

Comment:
Should match `DECLARE_AND_CHECK_HOST_ARCH`
```suggestion
  declare_configuration_metadata("architecture", "CPU architecture", "Milk-V (RISC-V)")
```

Reply:
It has been changed.